packages/schemas/tables/hooks.sql

```diff
@@ -1,13 +1,17 @@
 create table hooks (
-    tenant_id varchar(21) not null
-      references tenants (id) on update cascade on delete cascade,
-    id varchar(21) not null,
-    event varchar(128) /* @use HookEvent */ not null,
-    config jsonb /* @use HookConfig */ not null,
-    created_at timestamptz not null default(now()),
-    primary key (id)
-  );
+  tenant_id varchar(21) not null
+    references tenants (id) on update cascade on delete cascade,
+  id varchar(21) not null,
+  name varchar(256),
+  event varchar(128) /* @use HookEvent */,
+  events jsonb /* @use HookEvents */,
```

Comment:
after a preliminary search it looks like `events` [cannot be indexed easily](https://dba.stackexchange.com/questions/307484/create-an-index-for-fields-within-an-array-of-composite-type). if it is the case, we should create another table instead?